docs/snippets/modules/model_io/prompts/prompt_templates/get_started.mdx

```diff
@@ -43,7 +43,7 @@ there is a mismatch; for example,
 from langchain import PromptTemplate
 
 invalid_prompt = PromptTemplate(
```

Comment:
believe this is meant to be invalid

Reply:
i meant i believe this prompt is intentionally invalid for demonstration purposes